src/main/views/UserProfileView.java

```diff
@@ -0,0 +1,55 @@
+package views;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import constants.Sex;
+import controllers.UserProfileController;
+import java.text.SimpleDateFormat;
+import java.lang.Object;
+
+import dto.LoginDTO;
+import dto.ResponseDTO;
+import dto.SignUpDTO;
+import dto.UserProfileDTO;
+import exceptions.DataEmptyException;
+import exceptions.NotFoundException;
+import exceptions.UserProfileException;
+import models.users.UserAccount;
+
+public class UserProfileView extends View{
+    public UserProfileView(Connection con) {
+        super(con);
+    }
+
+    public ResponseDTO<Object> insertUserProfile (Connection con, UserProfileDTO user_profile, SignUpDTO sign_up) throws 
+                DataEmptyException, UserProfileException, NotFoundException, java.text.ParseException, SQLException {
+
+        String account_id = UserAccount.getIdByUsername(con, sign_up.getUsername());
+        ResponseDTO<Object> response = null;
+        user_profile.setAccountId(account_id);
```

Comment:
dont set accountId here! All logic equivalent to backend logic will be produce in service and lower.